src/cdk/coercion/number-property.ts

```diff
@@ -10,8 +10,13 @@
 export function coerceNumberProperty(value: any): number;
 export function coerceNumberProperty<D>(value: any, fallback: D): number | D;
 export function coerceNumberProperty(value: any, fallbackValue = 0) {
+  return isNumberValue(value) ? Number(value) : fallbackValue;
+}
+
+/** Whether the provided value is considered a number. */
+export function isNumberValue(value: any): boolean {
```

Comment:
I'm not sure this should be exposed as a public API under coercion. Mark this as internal?

Reply:
Marked as `docs-private`